src/containers/Application.js

```diff
@@ -124,11 +165,12 @@ class Application extends Component {
 		else return (
 			<Provider store={this.Store}>
 				<div className="app">
-					<Header app={this}/>
+					<Header hasSidebar={this.props.hasSidebar} app={this}/>
 					<div className="app-body">
-						<Sidebar app={this} navigation={this.Navigation}/>
+						<Sidebar hasSidebar={this.props.hasSidebar} app={this} navigation={this.Navigation}/>
```

Comment:
The actual use of sidebar can be conditional - not via `hasSidebar` property.

Reply:
Changed in last 2 commits